src/constructor.io/utils/Helpers.cs

```diff
@@ -163,6 +163,18 @@ protected static string MakeUrl(Hashtable options, List<string> paths, Hashtable
                     }
                 }
 
+                // Add quiz answers to query string
+                if (queryParams.Contains(Constants.ANSWERS))
```

Comment:
Nice catch, missed this one.

Looking good to me. Lets have someone else have a final pass.